### PR TITLE
isTun2socksRunning bool

### PR DESCRIPTION
**Pull request by rurirei**
more status check is needed, as #728

### CODE REVIEW

File: AndroidLibV2rayLite/CoreI/Status.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 
 type Status struct {
 	IsRunning       bool
+	IsTRunning      bool
```
Comment (floatingrain) [inline AndroidLibV2rayLite/CoreI/Status.go:9]: 命名不应该使用“意义不明确的缩写”，这点各大编码规范中都有提及。建议
```suggestion
	IsTun2socksRunning bool
```

File: AndroidLibV2rayLite/interact.go [hunk 2]
```diff
@@ -111,6 +111,10 @@ func (v *V2RayPoint) GetIsRunning() bool {
 	return v.status.IsRunning
 }
 
+func (v *V2RayPoint) GetIsTRunning() bool {
```
Comment (floatingrain) [inline AndroidLibV2rayLite/interact.go:114]: 同样的，建议完整拼写命名

**Review by floatingrain — CHANGES_REQUESTED** [review]
命名问题

File: AndroidLibV2rayLite/CoreI/Status.go (hunk 1 above)
Comment (rurirei) [reply AndroidLibV2rayLite/CoreI/Status.go:9]: do not use variable too long.

Comment (floatingrain) [reply AndroidLibV2rayLite/CoreI/Status.go:9]: > do not use variable too long.

Maybe "IsT2SRunning" is better?

**Comment by yuhan6665** [comment]
why do we need this bool? who will call GetIsTRunning()?

**Comment by rurirei** [comment]
> more status check is needed, as #728

you could do next to handle possible errors on Tun2socks start of escort.go. @yuhan6665

**Comment by yuhan6665** [comment]
> > more status check is needed, as #728
> 
> you could do next to handle possible errors on Tun2socks start of escort.go. @yuhan6665

#728 is a bit weird where looks like tun2socks error out after 10 seconds. Though I can't reproduce this issue. I think we need tun2socks log to know what happened in his device.
Even when there is an error in tun2socks, I think isTun2socksRunning will be true right?

**Comment by rurirei** [comment]
> > > more status check is needed, as #728
> > 
> > 
> > you could do next to handle possible errors on Tun2socks start of escort.go. @yuhan6665
> 
> #728 is a bit weird where looks like tun2socks error out after 10 seconds. Though I can't reproduce this issue. I think we need tun2socks log to know what happened in his device.
> Even when there is an error in tun2socks, I think isTun2socksRunning will be true right?

yes you need to generate error from v.runTun2socks(), as it returns nil always.

**Pull request merged**